kotlin.web.demo.frontend/src/application/Application.kt

```diff
@@ -387,12 +425,21 @@ object Application {
                     versionView.init(kotlinVersions)
                     kotlinVersions.forEach { wrapperConfig ->
                         val version = wrapperConfig.version
-                        iframeDialogs[version] = IframeDialog(version)
+                        availableKotlinVersions.add(version)
```

Comment:
Maybe availableKotlinVersions should be lateinit/nullable and instantiated here?